test/image/mocks/zz-text_on_shapes_texttemplate.json

```diff
@@ -212,16 +211,6 @@
                 },
                 "editable": true
             },
-            {
-                "label": {
-                    "texttemplate": "paths have no slope: %{slope}",
-                    "textposition": "top center",
-                    "yanchor": "bottom"
-                },
-                "editable": true,
-                "path": "M3.71,0.75L2.26,3.99L3.71,3.99Z",
-                "layer": "above"
-            },
```

Comment:
OK. Please update the baseline so that image test pass: https://app.circleci.com/pipelines/github/plotly/plotly.js/8520/workflows/491afb80-1892-4f25-8f62-b5b25db1bfb4/jobs/188422/artifacts